prow/prowjobs/google/jwt_verify_lib/jwt-verify-lib-presubmit.yaml

```diff
@@ -10,7 +10,7 @@ presubmits:
       description: "Runs all unit tests per PR."
     spec:
       containers:
-      - image: gcr.io/cloudesf-testing/gcpproxy-prow:v20220829-v2.38.0-5-gc746373d-master
+      - image: gcr.io/cloudesf-testing/gcpproxy-prow:v20221109-jwt-verify-lib
```

Comment:
instead of tag it as `jwt_verify_lib`,   maybe we should call it out in the name,  e.g. `gcr.io/cloudesf-testing/jwt-verify-lib-prow`

Reply:
That's a good idea. Done :)